src/tests/unit/tests/common/react/named-fc.test.tsx

```diff
@@ -9,20 +9,20 @@ describe('NamedFC', () => {
     const MyFC = NamedFC('MyFC', () => <span>TEXT</span>);
 
     it('applies correct displayName', () => {
-        expect(MyFC.displayName).toEqual('MyFC');
+        expect(MyFC.displayName).toMatchSnapshot('MyFC');
```

Comment:
This can stay as it was before.
```suggestion
        expect(MyFC.displayName).toEqual('MyFC');
```

Reply:
Made the change